app/shared/definition/ApiDefinition.scala

```diff
@@ -16,26 +16,30 @@
 
 package shared.definition
 
-import play.api.libs.json.{Format, Json, OFormat}
+import play.api.libs.json.*
 import shared.routing.Version
-import shared.utils.enums.Enums
 
-sealed trait APIStatus
+enum APIStatus {
+  case ALPHA, BETA, STABLE, DEPRECATED, RETIRED
+}
 
 object APIStatus {
-  val parser: PartialFunction[String, APIStatus] = Enums.parser[APIStatus]
-
-  case object ALPHA extends APIStatus
-
-  case object BETA extends APIStatus
-
-  case object STABLE extends APIStatus
-
-  case object DEPRECATED extends APIStatus
-
-  implicit val formatAPIStatus: Format[APIStatus] = Enums.format[APIStatus]
+  val parser: PartialFunction[String, APIStatus] = {
+    case status if APIStatus.values.exists(_.toString == status) =>
+    APIStatus.values.find(_.toString == status).get
+  }
 
-  case object RETIRED extends APIStatus
+  given Format[APIStatus] = Format(
+    Reads {
+        case JsString(status) => APIStatus.values.find(_.toString == status).map(JsSuccess(_))
+          .getOrElse(JsError(s"Unknown APIStatus: $status"))
+        case _ => JsError("Expected a string for APIStatus")
+    },
+    Writes {
+      (status: APIStatus) => JsString(status.toString)
+    }
+  )
+  
 }
```

Comment:
This can be cleaned up as there seems to be repetitions of this code for enums. The Enums utility should not have been deleted but updated to make it work with Scala 3 upgrade. See the comment on that deleted file. How it can be done can be seen [here](https://github.com/hmrc/individuals-reliefs-api/commit/8d5ec9fe06ffdad860876b9e2e7d50a544d5c0c4#diff-b084d4b508d1ffed46b2d5ffd812bd7239a99146ba3ede40892a89f31c466cb9) once that file is reinstated and works with the necessary changes, thanks.